### PR TITLE
Document pre and post-merge steps in `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by g1zzm0**
In this patch, we document what steps need to be undertaken before and
after a merge.

In particular, we highlight that branches need to be deleted since
lingering branches are confusing for the developers and prevent
various cache clean-ups in Git.

### CODE REVIEW

**Comment by g1zzm0** [comment]
@mristin I think im done now. All steps and motivation should be clear.

**Comment by mristin** [comment]
LGTM!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -30,26 +34,43 @@ The commit messages follow the guidelines from https://chris.beams.io/posts/git-
...
 
-To validate the example data against the schemas, call:
+Afterwards you run the script to validate the example data against the schemas by calling:
 
 ```
 schemas\Validate.ps1
 ```
 
+### Check Commit and Pull Request Messages
+In accordance with `Recommendation for Commit Messages` the continuous integration checks the previously defined conditions. For the present development, however, this is not enforced.
```
Comment (mristin) [inline CONTRIBUTING.md:70]: Need to be split in two lines, one sentence per line

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -93,3 +114,10 @@ Those assignees are:
 - [@aorzelskiGH]
 
 [@aorzelskiGH]: https://github.com/aorzelskiGH
+
+## Post-Merge Cleanup
+**Conratulaion.** You sucessfully contributed to the aas-spec repository.
+
+If you are a member of the development team, pleas delete the feature branch you directly created within the aas-specs repository.
+
+Otherwise, feel free to delete your forked ass-spec repository in your repositories.
```
Comment (mristin) [inline CONTRIBUTING.md:123]: "Otherwise, if you are not a team member and you forked the repository, ..."

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -30,26 +34,43 @@ The commit messages follow the guidelines from https://chris.beams.io/posts/git-
...
 
-## Pre-merge Checks
+## Create Pull Request
+After all changes have been comitted to your feature branch, a [pull request] has to be created. See [this Github tutorial] for more guidance. 
+
+[pull request]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request
+
+## Pre-Merge Checks
+**Continuous Integration**
+Github will run the continuous integration (CI) automatically through Github actions to verify that the submitted changes are valid. Every pull request automaticaly runs the continuous integration with every update.
```
Comment (mristin) [inline CONTRIBUTING.md:44]: Need to be split on two lines, one sentence per line

**Review by mristin — COMMENTED** [review]
Sorry, I noticed a couple of nitpicks after merging into master. I'll make these minor changes myself.

**Pull request merged**